src/scripts.c

```diff
@@ -556,6 +559,36 @@ static int kiavc_lua_method_getversionstring(lua_State *s) {
 	return 1;
 }
 
+/* Require version as major, minor, patch */
+static int kiavc_lua_method_requireversion(lua_State *s) {
+	/* This method allows the Lua script to check whether the engine is of a compatible version (and quit if not) */
+	int n = lua_gettop(s), exp = 3;
+	if(n != exp) {
+		SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "[Lua] Wrong number of arguments in requireVersion: %d (expected %d)\n", n, exp);
+		kiavc_cb->quit();
+		return 0;
+	}
+	int major = luaL_checknumber(s, 1);
+	if(major != KIAVC_VERSION_MAJOR) {
+        SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "[Lua] KIAVC major version number incompatible: %d (expected %d)\n", KIAVC_VERSION_MAJOR, major);
+        kiavc_cb->quit();
```

Comment:
Indentation seems wrong in this checks and the ones below? Looks like you're using spaces instead of tabs (which I'd rather we use in this project, since it's what I'm used to).

Reply:
Sorry, I'm used to the opposite, so I didn't notice, but it's fixed now.